main.go

```diff
@@ -88,6 +93,22 @@ func handler(w http.ResponseWriter, r *http.Request) {
 	log.Debugf("Scrape of target '%s' with module '%s' took %f seconds", target, moduleName, duration)
 }
 
+func updateConfiguration(w http.ResponseWriter, r *http.Request) {
```

Comment:
Look at how it's done in https://github.com/prometheus/blackbox_exporter/blob/master/main.go